style.css

```diff
@@ -1,101 +1,93 @@
-body,
-h1,
-h2,
-p {
-  margin: 0;
-  color: #091e42;
-}
-
-.nav-bar {
-  display: flex;
-  background-color: #3c3a39;
-  color: #fff;
-  height: 5.77em;
-  align-items: center;
-  padding-right: 1.26em;
-}
-
-.logoname {
-  font-family: "Inter", sans-serif;
-}
-
-.nav-bar a {
-  color: #fff;
-  font-size: 1.26em;
-  flex: 1;
-  display: flex;
-  margin: 0 2rem;
-  text-decoration: none;
-  font-family: "inter", sans-serif;
-}
-
-#menu {
-  justify-content: flex-end;
-  width: 1.125rem;
-  border: none;
-  background-color: transparent;
-  background-image: url(../images/menu.svg);
-  background-repeat: no-repeat;
-  background-position: center;
-}
-
-a.button,
-button {
-  padding: 0.76rem;
-  color: #fff;
-  background: none;
-  text-align: center;
-}
-
-.mobile-view {
-  width: 1.125rem;
-  border: none;
-  background-color: transparent;
-  background-image: url(../images/menu.svg);
-  background-repeat: no-repeat;
-  background-position: center;
-}
-
-.first-sec {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  padding: 0 11%;
-  row-gap: 0.8rem;
-  background-color: #1c1a19;
-  background-image: url(../images/background-mobile.png);
-  background-repeat: no-repeat;
-  background-position: bottom right;
-  background-size: auto 15.626em;
-  height: 38rem;
-  color: #fff;
-}
-
-.first-sec h1 {
-  color: #e05304;
-  font-size: 3.6rem;
-  font-family: "Crete Round", serif;
-}
-
-.first-sec h2 {
-  color: #fff;
-  font-size: 1.5rem;
-  line-height: 2.1rem;
-  font-weight: 700;
-  font-family: "Roboto", sans-serif;
-}
-
-.first-sec p {
-  color: #fff;
-  font-size: 1.2rem;
-  font-family: "Poppins", sans-serif;
-  line-height: 1.6rem;
-}
-
-.social-media-icons {
-  display: flex;
-  gap: 1.2rem;
-  padding: 0;
-  margin: 0;
-  list-style: none;
-}
+* {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+main {
+  background-image: url("header-shapes.png");
+  background-repeat: no-repeat;
+}
+
+header {
+  position: fixed;
+  background-color: #fff;
+  top: 0;
+  width: 375px;
+  height: 90px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+header a {
+  margin-top: 58px;
+  margin-left: 24px;
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  width: 76px;
+  height: 20px;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.fa-bars {
+  width: 24px;
+  height: 24px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  top: 8px;
+  margin-left: 180px;
+  color: #6070ff;
+}
+
+section {
+  margin: 128px 24px 136px 24px;
+}
+
+.section-title {
+  color: #172b4d;
+  font-family: "Poppins", sans-serif;
+  width: 327px;
+  height: 150px;
+  font-size: 40px;
+}
+
+p {
+  width: 327px;
+  height: 146px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
```

Comment:
This is the Figma design:
![image](https://user-images.githubusercontent.com/61361037/212133008-af650e9c-c76a-4223-b9c6-16c5c9fab5c0.png)

This is your design:
![image](https://user-images.githubusercontent.com/61361037/212133136-f27d8db1-1ad8-4e3d-9172-e32e98a530ed.png)

Remove this `width` and `height` values and let the elements decide their own space.
You have to use `display: flex` & `flex-direction: column` to align all the items vertically.
Then the elements will start aligning themselves without the use of hard-fix values of width, height, padding and margins.